repository/spark/docs/latest/installation.md

```diff
@@ -12,18 +12,29 @@ Requirements:
 Check out existing [limitations](limitations.md) before installing a KUDO Spark instance. Currently, multi-instance 
 (multi-tenant) operator installation supports only a single instance per namespace. 
 
+Create a namespace for the operator:
 ```
-kubectl kudo install spark --instance=spark
+kubectl create ns spark
+```
+To install a new instance of Spark operator from the official repository, use the following command:
+```
+kubectl kudo install spark --namespace spark
+```
+This will install Spark operator instance with a name `spark-instance` to the provided namespace.
+You can also specify instance name using `--instance` parameter:
```

Comment:
```suggestion
This will install a Spark operator instance with the name `spark-instance` to the provided namespace.
You can also specify a different instance name using `--instance` parameter:
```